src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/TempListView.container.js

```diff
@@ -1,23 +1,42 @@
 // @flow
 import { connect } from 'react-redux';
-import EventsListWrapper from './EventsListWrapper.component';
-import { makeColumnsSelector, makeCreateEventsContainer, makeCreateWorkingListData } from './eventsList.selector';
-import { sortWorkingList, openViewEventPage, requestDeleteEvent } from '../listView.actions';
-import { updateWorkinglistOrder } from './actions/columnSelectorDialog.actions';
+import withLoadingIndicator from '../../../../HOC/withLoadingIndicator';
+import withErrorMessageHandler from '../../../../HOC/withErrorMessageHandler';
+import { ListView } from '../../../ListView';
+import { makeColumnsSelector, makeCreateEventsContainer, makeCreateWorkingListData } from './tempListView.selector';
+import { sortWorkingList, openViewEventPage, requestDeleteEvent } from '../../../ListView/listView.actions';
+import { updateWorkinglistOrder } from '../../../ListView/ColumnSelector/actions/columnSelectorDialog.actions';
 
 const makeMapStateToProps = () => {
     const columnsSelector = makeColumnsSelector();
     const createEventsContainer = makeCreateEventsContainer();
     const createWorkingListData = makeCreateWorkingListData();
 
+    /* eslint-disable complexity */
     const mapStateToProps = (state: ReduxState, props: { listId: string }) => {
         const listId = props.listId;
-        const isLoading = !!state.workingListsUI[listId].isLoading;
-        const columns = !isLoading ? columnsSelector(state, props) : null;
+        const { isLoading, dataLoadingError } = state.workingListsUI[listId];
+
+        if (isLoading) {
```

Comment:
I am curious if we could avoid the complexity that eslint reports by returning default values when `isLoading` or `dataLoadingError` have values. 

We already return some default values for those cases eg
```
const sortById = !isLoading ? state.workingListsMeta[listId].sortById : null;
```

Maybe I dont understand the use-case well enough?

Reply:
As mentioned in the introduction this file will be deleted/refactored in future PR 🙂(Update: It will be deleted).

Reply:
Good stuff!